lib/celluloid/eventsource.rb

```diff
@@ -1,3 +1,4 @@
+require 'celluloid/current'
```

Comment:
These eliminate the celluloid backport warning-- orthogonal to the retry change, but thrown in this PR.